{
	"header": {
		"line": "CVE-2021-3597 (A flaw was found in undertow. The HTTP2SourceChannel fails to write th ...)",
		"name": "CVE-2021-3597",
		"description": "(A flaw was found in undertow. The HTTP2SourceChannel fails to write th ...)"
	},
	"annotations": [
		{
			"line": "\t- undertow 2.2.10-1 (bug #989861)",
			"type": "package",
			"package": "undertow",
			"kind": "fixed",
			"version": "2.2.10-1",
			"flags": [
				{
					"bug": 989861
				}
			]
		},
		{
			"line": "\tNOTE: https://bugzilla.redhat.com/show_bug.cgi?id=1970930",
			"type": "NOTE",
			"description": "https://bugzilla.redhat.com/show_bug.cgi?id=1970930"
		}
	]
}
